marlowe-playground-client/default.nix

```diff
@@ -0,0 +1,49 @@
+{ pkgs, set-git-rev, haskell, docs, easyPS, webCommon, nodejs-headers }:
```

Comment:
yep, we talked about this on slack. I went over nix/docs.nix and moved the expressions to the respective paper/doc directories in a localized default.nix so that it's consistent with this.